app/src/main/java/com/github/h3lp3rs/h3lp/dataclasses/MedicalInformation.kt

```diff
@@ -24,11 +24,13 @@ class MedicalInformation(
         require(size in MIN_HEIGHT..MAX_HEIGHT)
         require(weight in MIN_WEIGHT..MAX_WEIGHT)
         require(yearOfBirth in MIN_YEAR..Year.now().value)
-        try {
-            val number = PhoneNumberUtil.getInstance().parse(emergencyContactNumber, DEFAULT_COUNTRY)
-            require( PhoneNumberUtil.getInstance().isPossibleNumber(number))
-        } catch(e: Exception){
-            throw IllegalArgumentException(e.message)
+        if(emergencyContactNumber != ""){
```

Comment:
You can also extract this empty string into a const val with a meaningful name like: EMPTY_NB for example, since it was used often in the code

Reply:
Good idea!